11-functions.Rmd

```diff
@@ -119,7 +119,17 @@ make_shades <- function(colour, n, lighter = TRUE) {
 }
 ```
 
+#### Using your custom functions
+
+Now that you've created your custom functions there are a few ways you can load it in so it can be used by other notebooks and scripts. Depending on the situation you'll want to pick either `source`ing the script or creating an `R package`.
+
+- `source()` - This is most straightforward option. You can put this at the top of your notebook or script. It just involves you pointing to the script that holds your custom functions: `source(file/path/here.R)`. This is good for if you are going to reuse these custom functions mostly within one project. But may not be ideal if
```

Comment:
```
- `source()` - This is most straightforward option. To do this can put this at the top of your notebook or script. It just involves you pointing to the script that holds your custom functions: `source(file/path/custom_functions_file.R)`. This option is good if you are going to reuse these custom functions mostly within one project. However, it may not be ideal if you want to use your function across multiple projects. 
```